resources/runners/qemu.sh

```diff
@@ -1,3 +1,16 @@
+APPEND="root=/dev/sda ip=dhcp vga=0x344"
+while [ ! $# -eq 0 ]
+do
+    case "$1" in
+        --terminal | -t)
+            APPEND="$APPEND console=ttyAMA0,115200 console=ttyS0"
```

Comment:
I got it from here: http://stackoverflow.com/questions/19565116/redirect-qemu-window-output-to-terminal-running-qemu#comment42884295_19569340

Apparently some emulated machines use it. I can take it out if you think we should.